compute_rhino3d/Util.py

```diff
@@ -32,7 +32,9 @@ def default(self, o):
     if apiKey:
         headers['RhinoComputeKey'] = apiKey
     r = requests.post(posturl, data=postdata, headers=headers)
-    return r.json()
+    if r.headers.get('content-type').lower() == 'application/json':
+        return r.json()
+    raise Exception("Error from compute endpoint\n{}".format(r.text))
```

Comment:
you can use `f"Error from compute endpoint\n{r.text}"`

Reply:
That's python 3 right? Rhino uses python 2.7 internally but I wasn't sure about this package's version. Played it safe and stuck with `.format()`

Reply:
introduced in python 3.6

i saw 3.10 in your stack trace so i made the assumption =)

Reply:
If the maintainers say this package is python 3 safe i'll make the change